tkltest/util/constants.py

```diff
@@ -21,8 +21,9 @@
 
 TKLTEST_UNIT_OUTPUT_DIR_PREFIX = 'tkltest-output-unit-'
 
-# lib download dir
-TKLTEST_LIB_DIR = os.path.join('..', 'lib')
+# cli lib download dir
+TKLTEST_CLI_DIR = os.getcwd()
```

Comment:
- How will the lib directories be found if they are relative the CLI dir, which is set to current working dir (which could be anywhere in the filesystem?
- It would be better to call this dir `TKLTEST_HOME` or `TKLTEST_ROOT`; that’s the usual convention
- We need to think through how would the home/root dir work in for the installed `tkltest` commands (via pip install)

Reply:
tnx
we have the cli_dir, and the output dir (a directory per app, or per module if we have multi modules)
Currently, the user runs its command at the cli dir, so the line:
TKLTEST_CLI_DIR = os.getcwd()
will set TKLTEST_CLI_DIR .
the lib dir is always a sub directory of the cli dir, so:
TKLTEST_LIB_DIR = os.path.join(TKLTEST_CLI_DIR, 'lib')

Currently, the output dir is a sub-directory of the cli dir. (or sub-sub-sub-directory in case of modules)

I guess we would like to:
1. have a cli dir, and change its name to TKLTEST_HOME.
2. have a working dir - the directory in which the user runs the tkltest, and the tkltest output all its artifacts
3. an output dir - sub directory of working dir, per app 

however, it is a big change, not for this PR